totfront/somewhere-travel
# feat: basic file structure && css midules preset

### What was implemented?
1. Created a basic structure of a project
2. Created a showcase with CSS modules
3. Implemented the preparation for performance tracking and test coverage

### How to test it?
1. Run yarn start, build should not be failed
2. Check the files stucture
